frontend-react/src/components/DataDashboard/ReportDetails/ReportDetailsTable.tsx

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+
+import { useReportsFacilities } from "../../../hooks/network/DataDashboard/DataDashboardHooks";
+import Table, { TableConfig } from "../../../components/Table/Table";
+import { EventName, trackAppInsightEvent } from "../../../utils/Analytics";
+import TableFilters from "../../Table/TableFilters";
+import useFilterManager, {
+    FilterManagerDefaults,
+} from "../../../hooks/filters/UseFilterManager";
+import { FeatureName } from "../../../AppRouter";
+
+import styles from "./ReportDetailsTable.module.scss";
+
+const filterManagerDefaults: FilterManagerDefaults = {
+    sortDefaults: {
+        column: "collectionDate",
+        order: "DESC",
+    },
+};
+
+interface ReportDetailsTableProps {
+    reportId: string;
+}
+
+function ReportDetailsTable(props: ReportDetailsTableProps) {
+    const { reportId }: ReportDetailsTableProps = props;
+    const { data } = useReportsFacilities(reportId);
+    const featureEvent = `${FeatureName.REPORT_DETAILS} | ${EventName.TABLE_FILTER}`;
+
+    const filterManager = useFilterManager(filterManagerDefaults);
+
+    const tableConfig: TableConfig = {
+        columns: [
+            {
+                dataAttr: "provider",
+                columnHeader: "Ordering provider",
+                feature: {
+                    link: true,
+                    linkBasePath: "/data-dashboard/facilities-providers/",
+                },
+            },
+            {
+                dataAttr: "facility",
+                columnHeader: "Performing facility",
+                feature: {
+                    link: true,
+                    linkBasePath: "/data-dashboard/facilities-providers/",
+                },
+            },
+            {
+                dataAttr: "submitter",
+                columnHeader: "Submitter",
+                feature: {
+                    link: true,
+                    linkBasePath: "/data-dashboard/facilities-providers/",
+                },
+            },
+            { dataAttr: "total", columnHeader: "Test results" },
+            {
+                dataAttr: "collectionDate",
+                columnHeader: "Specimen collection date",
+            },
+        ],
+        rows: data!!,
```

Comment:
I noticed that the links in the new `report-detail` page are not URL safe. Do they need to be URI encoded using `encodeURIComponent`? Or some other way of concatenation? 

For example, locally, when I hover on this link, you can see on the bottom left of the page, the hyperlinked URL:

![Screenshot 2023-05-14 at 9 07 58 PM](https://github.com/CDCgov/prime-reportstream/assets/516877/49ba0836-b14a-4bdb-8266-5aad8ce239b1)

`http://localhost:3000/data-dashboard/facilities-providers/Any lab USA` is not valid.